### ISSUE TITLE
Returned RuleMatch contains null sentence 

### ISSUE THREAD

**Issue by MaciekKuna** [languagetool-org/languagetool#1707]
Hello, 
when the result is returned, the sentence attribute is missing (null). Below example of German sentence.

LanguageTool - 4.5

Example sentence:
unberechtigten Personen (Endnutzern) bekannt zu machen bzw. diese in cc zu setzen

![image](https://user-images.githubusercontent.com/16246708/61064249-b9c30880-a401-11e9-9e2f-26b8b7abbd0f.png)

**Comment by danielnaber** [comment]
Could you submit a small unit test to reproduce?

**Comment by MaciekKuna** [comment]
Hi, no problem.

```
        JLanguageTool spellchecker = new JLanguageTool(new GermanyGerman());
                spellchecker.disableRule("WHITESPACE_RULE");
                spellchecker.disableRule("DOPPELTE_SATZZEICHEN");

        List<RuleMatch> result =spellchecker.check("unberechtigten Personen (Endnutzern) bekannt zu machen bzw. diese in cc zu setzen");

        assertNotNull(result.get(1).getSentence().getText());
```

**Comment by danielnaber** [comment]
I cannot reproduce it with current master. It was maybe fixed after 4.5, probably the fix is also in 4.6.

**Comment by danielnaber** [comment]
Closing, as it's probably fixed now.
